src/main/resources/i18n/cs_CZ.yml

```diff
@@ -0,0 +1,506 @@
+        invalidSpawnPoint: "V konfiguracnim souboru je neplatny spawn point: {0}"
+        teamAdded: "Tym {0} byl nacten z konfiguracniho souboru"
+        namedTeamAdded: "Tym {0} ({1}) byl nacten z konfiguracniho souboru"
+        invalidTeam: "V konfiguracnim souboru je neplatny tym: {0}"
+        recipeApple: "Pridan novy recept na zlate jablko."
+        recipeCompass: "Pridan novy recept na kompas."
+        recipeMelon: "Pridan novy recept na zlaty meloun."
+        loaded: "UHC plugin byl nacten."
+
+        WBNotInstalled1: "{darkred}[UHC] {ce}Plugin WorldBorder neni nainstalovan - hranice sveta nemohou byt nastaveny!"
+        WBNotInstalled2: "{gray}Bez tohoto pluginu nelze hranice sveta nastavovat behem hry."
+        WBNotInstalled3: "{gray}Staci plugin nainstalovat, UHPlugin ho automaticky nakonfiguruje."
+
+        SPNotInstalled1: "{darkred}[UHC] {ce}Plugin SpectatorPlus neni nainstalovan."
+        SPNotInstalled2: "{gray}Pokud chcete spectator mod, staci plugin nainstalovat, UHPlugin ho automaticky nakonfiguruje."
+
+        PLNotInstalled1: "{darkred}[UHC] {ce}Potrebny ProtocolLib neni nainstalovan!"
+        PLNotInstalled2: "{gray}Nasledujici moznosti vyzaduji pritomnost ProtocolLib:"
+        PLNotInstalledItem: "{darkgray} - {gray}{0}"
+        PLNotInstalled3: "{gray}ProtocolLib je dostupny zde: {0}"
+
+    cmd:
+        errorLoad: "{ce}Vyskytla se chyba, vice informaci naleznete v konzoli. Pravdepodobne se jedna o bug."
+        errorUnknown: "{ce}Tento prikaz neexistuje. Napiste /uh pro seznam dostupnych prikazu."
+        errorUnauthorized: "{ce}lolnope"
+
+        titleHelp: "{yellow}{0} - verze {1}"
+        legendHelp: "{ci}Legenda: {cc}/uh prikaz <povinne> [volitelne=vychozi] <povolene mezery, ...>{ci}."
+        helpNextPage: "{gray}Dalsi stranka: /uh {0}"
+
+        titleGameCmd: "{aqua}------ Prikazy ke hre ------"
+        helpStart: "{cc}/uh start {ci}: spusti hru."
+        helpStartSlow: "{cc}/uh start slow {ci}: pomalu spusti hru ve dvou krocich (vhodne pro mensi servery)."
+        helpShift: "{cc}/uh shift {ci}: posune epizodu."
+        helpTeam: "{cc}/uh team {ci}: nastavuje tymy. Napiste /uh team pro vice detailu."
+        helpSpawns: "{cc}/uh spawns {ci}: nastavuje spawn points. Napiste /uh spawns pro vice detailu."
+        helpSpec: "{cc}/uh spec {ci}: nastavuje spectatory. Napiste /uh spec pro vice detailu."
+        helpWall: "{cc}/uh generatewalls {ci}: generuje zdi podle nakonfigurovanych parametru."
+        helpBorder: "{cc}/uh border {ci}: nastavuje hranice. Napiste /uh borders pro vice detailu."
+
+        titleBugCmd: "{aqua}------ Prikazy k administraci ------"
+        helpHeal: "{cc}/uh heal <player> [half-hearts=20|±diff] {ci}: uzdravi hrace o pocet zadanych pulsrdicek (standardne 20)."
+        helpHealall: "{cc}/uh healall [half-hearts=20|±diff] {ci}: uzdravi misto jednoho hrace vsechny."
+        helpFeed: "{cc}/uh feed <player> [foodPoints=20] [saturation=max] {ci}: nakrmi hrace."
```

Comment:
Well, if you really think it's better as-is, ok :) . I'll check all is OK tomorrow and merge.